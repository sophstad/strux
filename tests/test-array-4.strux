int main() {
  string[4] struxers = ["Josh", "Kofi", "Millie", "Sophie"];
  print(struxers[1]);
  struxers.show();

  bool[3] torf = [true, false, true];
  torf.show();
  return 0;
}
