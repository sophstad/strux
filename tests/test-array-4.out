Kofi
[Josh, Kofi, Millie, Sophie]
[1, 0, 1]
